### DISCUSSION TITLE
#Java

### DISCUSSION THREAD

**Discussion by hristijanjazhev** [codewars/codewars.com#2846] [Q&A]
After finding the solution i get stack at creating another class with main method in the editor...
So barely i have everything what i need , in my Eclipse it works fine,but when the part of calling the class and method in main method i get confused , i try to create another class in the class but doesn't work.

**Comment by hobovsky** [comment]
Coding in java you are in this good situation that it should be very easy for you to recreate codewars setup. You need one file with solution, one with tests, and run tests with junit5 runner. You dont need `main` or any additional code, just copy initial solution into one file, sample tests into another file, and you should be good to go.
